packages/pages/src/util/runSubprocess.test.ts

```diff
@@ -8,6 +9,10 @@ describe("runSubprocess", () => {
 
   it("returns non-zero when subprocess fails", async () => {
     const exitCode = await runSubprocess("echo", ["$((0/0))"]);
-    expect(exitCode).not.toEqual(0);
+    if (path.sep === path.posix.sep) {
+      expect(exitCode).not.toEqual(0);
+    }
+    // commands do not return exit codes on Windows, so we skip this test in that case
```

Comment:
Is this true? Wouldn't that mean any process we spawn that results in an exit code because we throw means it just hangs on Windows forever and never shuts down?

Reply:
It exits with code 0, so there is a code, but it doesn't get set.
It might be getting set in "errorlevel"

The command ${echo (0/0)} will produce an exit code of 1 in MacOs but 0 in Windows.

Reply:
This is because you can't execute a command during echo in Windows. It just returns it as a string. Try doing this:

`await runSubprocess("Set", ["/a", "foo=0/0"]);`

This will fail in unix because `Set` isn't a command. In Windows it will also fail with a divide by 0 error.

Reply:
It looks like it's actually that the echo command doesn't return the exit code on Windows when it tries to print $((0/0)), even though there is an error inside. On MacOS the error gets bubbled up.

I changed the test to run a powershell command that will throw an error with exit code when on Windows.